easybuild/easyconfigs/a/autopep8/autopep8-1.4.4-intel-2018a-Python-3.6.4.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'PythonPackage'
+
+name = 'autopep8'
+version = '1.4.4'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = "https://github.com/hhatto/autopep8"
+description = """A tool that automatically formats Python code to conform to the PEP 8 style guide."""
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['4d8eec30cc81bc5617dbf1218201d770dc35629363547f17577c61683ccfb3ee']
+
+dependencies = [
+    ('Python', '3.6.4'),
+    ('pycodestyle', '2.5.0', versionsuffix),
+]
+
```

Comment:
@SimonPinches please add:
```
download_dep_fail = True
use_pip = True
sanity_pip_check = True
```

Reply:
@smoors - sorry for the delay.  I finally found time to do this.  Hope all's good now :-)